apps/site/lib/site_web/templates/trip_plan/_fare_calculator.html.eex

```diff
@@ -66,25 +66,27 @@
 </div>
 
 
-<div class="m-trip-plan-farecalc__table">
-  <div class="m-trip-plan-farecalc__table-cell m-trip-plan-farecalc__subheader">
-    Monthly Passes
-  </div>
-  <div class="m-trip-plan-farecalc__table-cell m-trip-plan-results__itinerary-fare-title m-trip-plan-farecalc__header">
-    Base
-  </div>
-  <div class="m-trip-plan-farecalc__table-cell m-trip-plan-results__itinerary-fare-title m-trip-plan-farecalc__header">
-    Reduced
-  </div>
-  <div class="m-trip-plan-farecalc__table-cell m-trip-plan-results__itinerary-fare-title m-trip-plan-farecalc__header">
-    Valid On
-  </div>
+<%= if @show_monthly_passes? do %>
```

Comment:
You might not need to compute and pass in a `show_monthly_passes?` value if you use the view function in the template.

```suggestion
<%= if SiteWeb.TripPlanView.show_monthly_passes?(@itinerary) do %>
```